### DISCUSSION TITLE
Suggestion: Add "Scan Library Files" option to Library context menu

### DISCUSSION THREAD

**Discussion by millercentral** [pannal/plex-for-kodi#111] [Ideas]
Now that we have the cool Library context menu, maybe add a "Scan Library Files" option for the library to this menu?  

Maybe needs to only be added for users that are the owner/administrator of the server... so might not be viable if that is not obvious to PM4K, but occasionally I need to do this when PMS doesn't notice file changes on the server and this would be convenient.

**Comment by pannal** [comment]
That's been done :)
